includes/class-freemius.php

```diff
@@ -8488,19 +8566,27 @@ function has_secret_key() {
          * @author Vova Feldman (@svovaf)
          * @since  1.0.9
          *
+         * @param string|bool $premium_suffix
+         *
          * @return string
          */
-        function get_plugin_name() {
+        function get_plugin_name( $premium_suffix = false ) {
             $this->_logger->entrance();
 
-            if ( ! isset( $this->_plugin_name ) ) {
+            if (
+                ! isset( $this->_plugin_name ) ||
+                (

```

Comment:
I don't think there's a point for this part of the `if` since once `$this->_plugin_name` is set, we never unset it.